keyboards/tweetydabird/lotus58/atmeldfu/info.json

```diff
@@ -0,0 +1,184 @@
+{                                                                                                 
+    "manufacturer": "Tweetys Wild Thinking",                                                      
+    "keyboard_name": "Lotus 58 Glow (QMK)",                                                             
+    "maintainer": "TweetyDaBird",                                                                 
+    "bootloader": "qmk-hid",                                                                      
+    "bootloader_instructions": "Short marked pads on PCB, or hold top-outer key when plugging in each hand.",
+    "bootmagic": {                                                                                
+        "enabled": true,                                                                          
+        "matrix": [0, 0]                                                                          
+    },                                                                                            
+    "build": {                                                                                    
+        "lto": true                                                                               
+    },                                                                                            
+    "debounce": 5,                                                                                
+    "diode_direction": "COL2ROW",                                                                 
+    "encoder": {                                                                                  
+        "rotary": [                                                                               
+            {                                                                                     
+                "pin_a": "F5",                                                                    
+                "pin_b": "F4",                                                                    
+                "resolution": 2                                                                   
+            }                                                                                     
+        ]                                                                                         
+    },                                                                                            
+    "features": {                                                                                 
+        "audio": false,                                                                           
+        "auto_shift": true,                                                                       
```

Comment:
```suggestion
```

Reply:
This will remain, bot in (keymap level) info.json and keymap.
Reason being the function should be there and compile in, but I do not want it toggled on at startup.